app/src/main/java/com/hungngo/themusicapp/base/DataBindingAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package com.hungngo.themusicapp.base
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+
+abstract class DataBindingAdapter<T>(diffCallback: DiffUtil.ItemCallback<T>) :
```

Comment:
```suggestion
abstract class BaseListAdapter<T>(diffCallback: DiffUtil.ItemCallback<T>) :
```
Cái này có liên quan tới databinding đâu ông :v